test/form-layout.html

```diff
@@ -545,6 +545,24 @@
         expect(estimateEffectiveColspan(layout.children[0])).to.be.closeTo(2, .1);
       });
 
+      it('should update style after layout-item is not hidden', async() => {
+        sinon.spy(layout, '_invokeUpdateStyles');
+        const firstDisplay = layout.firstElementChild.style.display;
```

Comment:
This is always empty (the element doesn't have an inline style for display), no need to store it

Reply:
changed test